addons/crm_project_issue/wizard/crm_lead2projectissue_wizard.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+
+class CrmLeadToProjectIssueWizard(models.TransientModel):
+    """ wizard to convert a Lead into a Project Issue and move the Mail Thread """
+    _name = "crm.lead2projectissue.wizard"
+    _inherit = 'crm.partner.binding'
+
+    lead_id = fields.Many2one('crm.lead', string='Lead', domain=[('type', '=', 'lead')], default=lambda self: self.env.context.get('active_id'))
```

Comment:
is this field required ? cannot we remove this field ? we can directly get lead_id from context with active_id of lead.even not displayed in view side..used here to just take value from lead which we directly get from context.get('active_id').